deployment/wsgi/gunicorn.conf.py

```diff
@@ -1,9 +1,10 @@
 wsgi_app = "bin.settings:application"
 bind = "0.0.0.0:8000"
-workers = 1
-preload_app = True
+workers = 2
+preload_app = False
 user = "gen3"
 group = "gen3"
 timeout = 300
-keepalive = 2
-keepalive_timeout = 5
+graceful_timeout = 45
+keepalive = 10
+pidfile = "/sheepdog/gunicorn.pid"
```

Comment:
could/should these changes be moved to a different PR?

Reply:
I think in terms of bigger picture all of them are solving a single problem of solving sheepdog crashes. But I get it, i can put that as a different PR to avoid confusion in the future.